cmd/gogio/help.go

```diff
@@ -59,4 +59,8 @@ The -work flag prints the path to the working directory and suppress
 its deletion.
 
 The -x flag will print all the external commands executed by the gogio tool.
+
+The -signkey flag specifies the path of the keystore, used for signature for Android. 
```

Comment:
Nit:

The -signkey flag specifies the path of the keystore, used for signing Android apk files.